tests/basilisp/testrunner_test.py

```diff
@@ -263,6 +265,35 @@ def test_fixtures_with_errors(
     result.assert_outcomes(passed=passes, failed=failures, errors=errors)
 
 
+def test_basilisp_test_noargs(pytester: pytest.Pytester):
+    runtime.Namespace.remove(sym.symbol("a.test-path"))
+
+    code = """
+    (ns tests.test-path
+      (:require
+       [basilisp.test :refer [deftest is]]))
+    (deftest passing-test
+      (is true))
+    """
+    pytester.makefile(".lpy", **{"./tests/test_path": code})
+
+    # I couldn't find a way to directly manipulate the pytester's
+    # `sys.path` with the precise control needed by this test, so we're
+    # invoking `basilisp test` directly as a subprocess instead ...
+    basilisp = shutil.which("basilisp")
+    cmd = [basilisp, "test"]
+    result = subprocess.run(cmd, capture_output=True, text=True, cwd=pytester.path)
```

Comment:
Does it not use the same `sys.path` as the running executable? By which I mean are you not able to monkeypatch `sys.path` to the value you need?

Reply:
Unfortunately, there's no way to replicate the desired behavior exactly. We need a test case where sys.path starts with `""`, followed by the minimal system Python paths. I couldn't achieve this using `pytester.runpytest` with `monkeypatch` or  [pytester.runpytest_subprocess](https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest.Pytester.runpytest_subprocess), as the latter always adds the test CWD to the path